deploy/ui/form.json

```diff
@@ -516,6 +516,13 @@
                       "description": "Reference object Name for the GitHooks."
                     }
                   ]
+                },
+                {
+                  "label": "SSH Secret",
+                  "type": "text",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.console.gitHooks.sshSecret",
+                  "description": "Secret to use for ssh key(s) mount."
```

Comment:
The (s) is incorrect as we only use one key. Suggestion:
Secret to use for ssh key and known hosts file